src/elements/hero/src/stories.tsx

```diff
@@ -200,11 +201,70 @@ MoneyHomepageHero.story = {
   }
 }
 
+const bbDealsHero = {
+  img: require('../../../../static/hero/mTilted.svg')
+}
+const breadcrumbs = (
+  <Breadcrumbs
+    crumbs={crumbs}
+    title="Understanding energy bills and electricity bills - FAQs and more"
+    variant="light"
+  />
+)
+const image = <img src="https://placekitten.com/200/200?image=9" alt="Salman" />
+const bottomImageOverflow = '33%'
+
+export const BBDealsExample = () => (
+  <div sx={{ height: 1200 }}>
+    <Hero
+      container={Container}
+      fgImage={bbDealsHero.img}
+      breadcrumbs={breadcrumbs}
+      bottomImageOverflow={bottomImageOverflow}
+    >
+      <div
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: 'column',
+          height: 'fit-content',
+          paddingBottom: bottomImageOverflow
+        }}
+      >
+        <Styled.h1
+          sx={{
+            fontSize: ['lg', 'xl'],
+            color: 'pure-white',
+            marginTop: 0,
+            maxWidth: ['80%', '60%']
+          }}
+        >
+          Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in
+          laying out print, graphic or web designs.
+        </Styled.h1>
+        <Author
+          variant="primary"
+          title="Jane smith"
+          titleUrl="https://www.jonathanfielding.com/"
+          authorImage={image}
+          subtitle="Broadband Deals Expert"
+        />
+      </div>
+    </Hero>
+  </div>
+)
+
+BBDealsExample.story = {
+  parameters: {
+    percy: { skip: true }
+  }
+}
 // @todo test with more than one image and position
 export const AutomatedTests = () => {
   return (
     <AllThemes themes={['uswitch', 'money']}>
       <ExampleWithKnobs />
+      <BBDealsExample />
```

Comment:
This has added the bbdeals component to each of the brands

Can we add themes={['uswitch', 'money', 'broadbanddeals']} instead?